doc/tor.md

```diff
@@ -146,6 +145,52 @@ password` (refer to the [Tor Dev
 Manual](https://2019.www.torproject.org/docs/tor-manual.html.en) for more
 details).
 
+## Example: creating an onion service on Debian/Ubuntu systems
```

Comment:
I would say you should move this to the bottom of the file as the final section. I would suppose the guide is for someone who has never set up a tor service for bitcoin or has very little experience using tor. Moving it to the bottom would mean this user has, hopefully, read the doc from top to bottom. Then, they can get on setting it up with an example.

Reply:
I would also refactor this whole section:
- Current bullet formatting looks akward
- Change the title so that a common naming format can be used if we want to add a guide for another system
  - From: `Example: creating an onion service on Debian/Ubuntu Systems`
  - To: `Setup Guide: Debian/Ubuntu`
- Better explanations per each step 

Here is how that would look. This is just a rough suggestion, so make contextual additions as you see fit:

## Setup Guide: Debian/Ubuntu

Assuming you already have an installation of Bitcoin Core:

##### 1. Install Tor:

To install Tor, run:

``` bash
sudo apt install tor
```

##### 2. Edit Tor configuration file:

open up the `torrc` file with your favorite text editor, this example uses the `nano` text editor:

``` bash
sudo nano /etc/tor/torrc
```

Edit the `torrc` file so that it has these settings:

```
ControlPort 9051
CookieAuthentication 1
CookieAuthFileGroupReadable 1
```

##### 3. Add user to Tor group:

``` bash
sudo usermod -a -G debian-tor satoshi
```

##### 4. Restart Tor

To restart Tor, run:

```
sudo systemctl restart tor
```

##### 5. Edit Bitcoin configuration file

Open up the `bitcoin.conf` file with your favorite text editor.
Edit the file so that it contains these settings:

```
listen=1
torcontrol=127.0.0.1:9051
debug=tor
```

##### 6. Confirmation
You should be all set up to run a node through the Tor Network.
You can confirm that everything is setup correctly by running the `getnetworkinfo` 
RPC command. The result should look something like this:

```
localaddresses": [
  {
    "address": "omy7kj7zwvfg5luayideh73uqb2latkoyyy5h65y4atv3fymnlxlzwqd.onion",
    "port": 8333,
    "score": 4
  }
]
```

Reply:
@jarolrod Made changes in https://github.com/bitcoin/bitcoin/pull/21157/commits/db59179fbddfb6cfb08365398fe53e862e57d845